packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -176,7 +176,13 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
     onDragEnd: _onDragEnd,
   };
 
-  const _onBackspace = (ev: React.KeyboardEvent<HTMLDivElement>) => {
+  const _onKeyDown = (ev: React.KeyboardEvent<HTMLDivElement>) => {
+    // Allow the caller to handle the key down
+    if (props.onKeyDown) {
+      props.onKeyDown(ev);
+    }
+
+    // Handle delete of items via backspace
     if (ev.which !== KeyCodes.backspace) {
       return;
```

Comment:
i think rather than doing this, since now we are working on keydown instead of just backspace, we should check if the keyboard event is backspace then we remove the elements. rather than returning if the keyboard event is not backspace.

Reply:
Good point